scss/action-buttons/_layout.scss

```diff
@@ -19,6 +19,17 @@
         .k-button + .k-button {
             border-left-width: 1px;
         }
+
+        .k-rtl &,
+        [dir="rtl"] & {
+            .k-button + .k-button {
+                border-left-width: 0;
+                border-right-width: 1px;
+                border-right-style: solid;
+                margin-left: 0;
+                border-radius: 0 0 0 $border-radius;
```

Comment:
You are right - by design there isn't a border-radius. I will remove this declaration